mdsets/core/base/BaseConnector.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+
+
+class BaseConnector(ABC):
+    """
+    Class intended to define the downloading and uploading mechanisms that
```

Comment:
All lines of code must be compliant with Black formatting rule: `line-length=88`.
Fixed already ✔️